### PR TITLE
Run tests using tfe.LocalConfig

### PR DESCRIPTION
**Pull request by mortendahl**
Tests where previously running using a `tfe.RemoteConfig` but that is fragile due to process starting and concurrency. This patch changes the tests to use `tfe.LocalConfig` instead.

Closes https://github.com/OpenMined/PySyft/issues/2211.

### CODE REVIEW

File: syft/frameworks/keras/model/sequential.py [hunk 1]
```diff
@@ -58,7 +58,6 @@ def share(model, *workers, target_graph=None):  # pragma: no cover
 
     # Push and initialize shared model on servers
     sess = tfe.Session(graph=target_graph)
-    tf.Session.reset(sess.target)
```
Comment (robert-wagner) [inline syft/frameworks/keras/model/sequential.py]: Should we add coverage back to this function?

File: test/keras/test_sequential.py [hunk 2]
```diff
@@ -43,7 +43,6 @@ def test_instantiate_tfe_layer():
     np.testing.assert_allclose(actual, expected, rtol=0.001)
 
 
-@pytest.mark.skip(reason="Currently breaks on macos")
 def test_share():  # pragma: no cover
```
Comment (robert-wagner) [inline test/keras/test_sequential.py]: We should remove the no cover pragma if we are running this test again

**Comment by robert-wagner** [comment]
You need to run `black syft` to get this pr to pass travis. Alternatively by running `make install_hooks` black will run automatically before every commit

File: syft/frameworks/keras/model/sequential.py (hunk 1 above)
Comment (mortendahl) [reply syft/frameworks/keras/model/sequential.py]: What do you mean here?

Comment (robert-wagner) [reply syft/frameworks/keras/model/sequential.py]: at the top of the function there is a comment of "#pragma: no cover" which prevents coverage from being run on this function. We should remove that if this function is now being tested

Comment (mortendahl) [reply syft/frameworks/keras/model/sequential.py]: Gotcha, and done!

Comment (robert-wagner) [reply syft/frameworks/keras/model/sequential.py]: Did you push your changes?

Comment (mortendahl) [reply syft/frameworks/keras/model/sequential.py]: jup

**Pull request merged**